### PR TITLE
removed duplicate cookiestore function

### PR DESCRIPTION
**Pull request by Myrinw**
## What kind of change does this PR introduce?

Docs update

## What is the current behavior?

In the developer guide for "Use Supabase with Next.js" we import the cookies function from next/headers.  ([On this page](https://supabase.com/docs/guides/auth/auth-helpers/nextjs))
Then we use it to setup a supabaseClient, however we should not do this since the imported createClient function from @/utils/supabase/server already uses the next-cookies to create this client. 

The default exported createClient from the utils folder included in the with-subapase next.js starter kit doesn't expect an argument, resulting in an typescript error if a user would follow this guide.

The issue related to this PR:
[https://github.com/supabase/supabase/issues/21175](The issue)

## What is the new behavior?
Now the cookieStore has been removed from this documentation and we no longer pass it into the createClient.

Updated version of the docs
docs-updated

## Additional context
The typescript error that a user would get following the current documentation:
Scherm­afbeelding 2024-02-10 om 19 01 49

The correct code the user would get after this documentation update:
Scherm­afbeelding 2024-02-10 om 19 01 55

### CODE REVIEW

**Review by encima — APPROVED** [review]

**Review by encima — APPROVED** [review]

**Comment by Myrinw** [comment]
Hey @encima , 
Sorry for re-request reviews:) it seems like I was unable to merge because the master branch was ahead. 
Not sure if it's possible but feel free to merge.

**Review by encima — APPROVED** [review]

**Comment by Myrinw** [comment]
@encima 
It seems  vercel  needs an approval for this fork deployment: 

Scherm­afbeelding 2024-02-13 om 08 13 43

**Comment by encima** [comment]
@Myrinw yep, all external PRs need approval from someone on the team before merging. Do not worry, I will approve the running and merge once passed. You have done the work of opening the PR so all is good from your side!

**Pull request merged**
